phoenix-shared/src/iosMain/kotlin/fr/acinq/phoenix/utils/LightningExposure.kt

```diff
@@ -417,3 +424,11 @@ suspend fun Peer.betterPayOffer(
     send(PayOffer(paymentId, payerKey, payerNote, amount, offer, fetchInvoiceTimeoutInSeconds.seconds))
     return res.await()
 }
+
+fun fakeWallet(amount: Satoshi, swapInParams: SwapInParams): WalletState.WalletWithConfirmations {
```

Comment:
This does not seem to be used anywhere, so it can be removed, or moved to the test module if needed.